### PR TITLE
Issue52 : 	First method change for approuval

### PR DESCRIPTION
**Pull request by emmurd**

### CODE REVIEW

**Comment by mat128** [comment]
Good job tackling #52 =)

With all the tests using the new method name, I'm worried that using remove_access_vlan does not behave the way unset_access_vlan does. If you remove remove_access_vlan from switch base, I'm sure all the tests still pass. Also, I would like to know that a certain call is deprecated when I call it. Maybe a log message or something?

**Comment by idjaw** [comment]
LGTM. Once conflicts are resolved, will merge.

**Comment by fbouliane** [comment]
Weird merge Merge branch 'master' into issue52 …?

File: tests/core/objects/backward_compatible_switch_operations_test.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from unittest import TestCase
+
+from flexmock import flexmock
+from netman.core.objects.switch_base import SwitchOperations
+
+
+class BackwardCompatiblieSwitchOperationsTest(TestCase):
```
Comment (idjaw) [inline tests/core/objects/backward_compatible_switch_operations_test.py]: Typo. I think it should be: BackwardsCompatibleSwitchOperationsTest

Comment (emmurd) [inline tests/core/objects/backward_compatible_switch_operations_test.py]: Oups!! Sorry! Fixing this right away!

**Comment by lindycoder** [comment]
:+1:

**Pull request merged**